### PR TITLE
Open graph

### LINKED ISSUE #60: opengraph tags
we want our website to look nice when embedded, https://ogp.me/

### PR DESCRIPTION
**Pull request by ezhang04**
Created and Added OpenGraphCommon to each file on website. Will add to other files once they are created
closes #60

### CODE REVIEW

**Review by ezrizhu — DISMISSED** [review]

**Comment by ezrizhu** [comment]
resolve build failure and this should be good

File: src/_includes/_layouts/About.jsx
Comment (ezrizhu) [inline src/_includes/_layouts/About.jsx:1]: This should be in `<head>`

**Review by ezrizhu — CHANGES_REQUESTED** [review]

**Comment by ezhang04** [comment]
image

**Review by ezrizhu — APPROVED** [review]
lg2m

**Pull request merged**